1014-input-design-guidance.md

```diff
@@ -0,0 +1,24 @@
+---
+category: component design guidance
+state: draft
+created: 2024-10-15
+updated: 2024-10-21
+---
+
+# Input Design Guidance
+
+[Input](https://clarity.design/documentation/input) component enable the user to input a single sentence or less of text.
+
+## Guidance
+
+- **Must** use Input when you need to allow user to enter a single sentence or less of text.
+- **Must** NOT use Input in case of multiple lines of text. Use [textarea](https://clarity.design/documentation/textarea) component to support multiple lines of text input.
```

Comment:
```suggestion
- **Must not** use an input in case of multiple lines of text. Use a [textarea](https://clarity.design/documentation/textarea) for multiple lines of text input.
```